nixos/modules/services/networking/unifi.nix

```diff
@@ -41,6 +41,30 @@ in
       '';
     };
 
+    services.unifi.jre = mkOption {
+      type = types.package;
+      default = pkgs.jre8;
+      description = ''
+        The JRE package to use. Check the release notes to ensure it is supported.
+      '';
+    };
+
+    services.unifi.unifi = mkOption {
+      type = types.package;
+      default = pkgs.unifiLTS;
```

Comment:
You need to add `defaultText = "pkgs.unifiLTS";` to make sure that the manual does not depend on `unifiLTS` (which is the error in the PR CI status), same for the other package options.

Reply:
Thank you for pointing that out! I did not know about `defaultText`.